db/migrate/20160203212001_add_card_token_to_order.rb

```diff
@@ -1,5 +0,0 @@
-class AddCardTokenToOrder < ActiveRecord::Migration
-  def change
-    add_column :orders, :card_token, :string
-  end
-end
```

Comment:
I'm curious why you deleted this. You almost never want to delete a migration, once it's been committed to master. Instead, make a new migration that deletes the column.